lib/formtastic/helpers/inputs_helper.rb

```diff
@@ -350,12 +350,21 @@ def association_columns(*by_associations) # @private
         end
       end
 
+      # Collects all foreign key columns
+      def foreign_key_columns
```

Comment:
I guess this should be marked as ` # @private` to not show in the yard documentation